src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -27,11 +28,16 @@ internal class FunctionExecutor : IFunctionExecutor
         private readonly IWebJobsExceptionHandler _exceptionHandler;
         private readonly TraceWriter _trace;
         private readonly IAsyncCollector<FunctionInstanceLogEntry> _fastLogger;
+        private readonly ILogger _logger;
```

Comment:
ILogger has a different design than TraceWriter so you have to think about it a little differently. Every unique category gets its own logger. It's different than TraceWriter where each Trace can get its own category and you only use one TraceWriter.

So one of these logs Result details -- which are unique and handled special and require their own category -- and one is for the Executor itself.